cratetorrent-cli/Cargo.toml

```diff
@@ -10,11 +10,11 @@ edition = "2018"
 
 [dependencies]
 cratetorrent = { path = "../cratetorrent" }
-flexi_logger = "0.16"
+flexi_logger = "0.18"
 futures = "0.3"
 hex = "0.4"
 log = "0.4"
 structopt = "0.3"
 termion = "1.5"
-tokio = { version = "0.2", features = ["fs", "io-std", "macros", "rt-threaded"] }
-tui = "0.13"
+tokio = { version = "1.9", features = ["macros", "rt-multi-thread", "io-std", "fs"] }
```

Comment:
I think we can just use the major version. If `tokio` follows SemVer, there shouldn't be backwards compatibility breakage with minor updates.